.run/Debug Secrets Provider.run.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Debug Secrets Provider" type="GoRemoteDebugConfigurationType" factoryName="Go Remote" port="43210">
+    <method v="2" />
+  </configuration>
+</component>
```

Comment:
can you please add a newline?

Reply:
No can do, this file is generated by the GoLand IDE.